lib/utils/group-helper.js

```diff
@@ -1,6 +1,22 @@
 const { resolve, join } = require('path');
 const { existsSync } = require('fs');
 
+const parseAndValidatedFilename = pathToResolve => {
+    const pathToArray = pathToResolve.split('/');
+    let pathBeforeFileName;
+    let fileName = pathToArray[pathToArray.length - 1];
```

Comment:
```suggestion
    let fileName = pathToArray.slice(-1).pop();
```
This seems to be much concise.

Reply:
a somewhat similar opinion like this https://github.com/webpack/webpack-cli/pull/1074#discussion_r327230213

Reply:
Anyway, the functional approach is good way to refactor. If you want then I guess I will change it! 😃